src/actionTypes.js

```diff
@@ -0,0 +1,2 @@
+export const LOGIN_USER_SUCCESS = 'LOGIN_USER_SUCCESS';
+export const LOGIN_USER_FAILED = 'LOGIN_USER_FAILED';
```

Comment:
This file should be shifted from the src folder to the actions folder.

Reply:
These files where shifted. Though there is no such convention.

Reply:
Noted. The file has been moved.